docs/plugins/program_output.js

```diff
@@ -24,7 +24,7 @@ const defaultOptions = {
     docsDir: './docs',
     sourceDir: './docs/sources',
     include: ['**.mdx', '**.md'],
-    commandPrefix: '',
+    commandPrefix: 'RASA_TELEMETRY_ENABLED=false',
```

Comment:
could you add it in the scripts/ folder instead?

https://github.com/RasaHQ/rasa/blob/8d4886a29623be4d703378bfc1a5e3f462dd364d/docs/scripts/compile_program_outputs.js#L7

Reply:
oh ho, gotcha, yes. makes more sense! Thanks 👍